packages/docs-ui/src/commands/commands/clipboard.inner.command.ts

```diff
@@ -101,7 +104,7 @@ export const InnerPasteCommand: ICommand<IInnerPasteCommandParams> = {
         if (docDataModel == null || originBody == null) {
             return false;
         }
-        const selections = originSelections.map((range) => BuildTextUtils.selection.getInsertSelection(range, originBody));
+        const selections = originSelections.map((range) => range);
```

Comment:
What's the purpose of this? If you would like to copy this array, there are more explicit ways.